pkg/contexts/ctxlog/ctxlog.go

```diff
@@ -4,17 +4,25 @@ import (
 	"context"
 
 	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/log/multislogger"
 	"go.opencensus.io/trace"
 )
 
 type key int
 
-const loggerKey key = 0
+const (
+	loggerKey  key = 0
+	sloggerKey key = 1
```

Comment:
I'm not sure we should pass slogger through context. 

If the contextual cues come from using `DebugContext(ctx, ...)` then carrying slogger in context feels off.

I might be wrong. Was there somewhere that felt better with this?

Reply:
If it's just to pass it to runLauncher, I'd change the signature.